.github/workflows/_undo.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  workflow_call:
+
+jobs:
+  undo:
+    runs-on: ubuntu-latest
+    if: ${{ always() && failure() && github.event_name == 'push' && contains(github.event.head_commit.message,'undo_true')}}
+    steps: 
+      - name: Checkout
```

Comment:
When does this run? Needs a comment to say

Reply:
This reverts a merge, if after merging it fails CI. I think there was some discussion a few months ago whether this should be automatic or not and it was decided to add a condition that the text "undo_true" is in the merge request title. 

However we actually use the merge request titles for Changelog entries now, so may not be the best place to add this... Is there a better place to put this flag. If this job isn't just automatic, I'm not certain it would be used... @coretl do you have thoughts on this?

Reply:
Personally I would delete this functionality, then "undo" the merge using the GitHub UI if it failed after merge...